nni/retiarii/integration.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import threading
+from typing import *
+
+import json_tricks
+import nni
+from nni.runtime.msg_dispatcher_base import MsgDispatcherBase
+from nni.runtime.protocol import send, CommandType
+from nni.utils import MetricType
+
+from . import utils
+from .graph import MetricData
+
+
+_logger = logging.getLogger('nni.msg_dispatcher_base')
```

Comment:
Why this logger?

Reply:
It should be removed. I added it for debug.

Reply:
To print message to dispatcher log? Any suggestions?